### ISSUE TITLE
Is anyone can share your wav2vec2.0 pretrain log to me? 

### ISSUE THREAD

**Issue by Gavin90s** [pytorch/fairseq#3391]
## Is anyone can share a full wav2vec2.0 pretrain log to me? Just want to know whether the wav2vec2.0 pretrain process is on the right way.
Any help will be greatly appreciated!

[2021-03-24 14:33:12,269][train_inner][INFO] - {"epoch": 2, "update": 1.946, "loss": "3.449", "ntokens": "6249.34", "nsentences": "77.8", "prob_perplexity": "25.864", "code_perplexity": "25.847", "temp": "1.791", "loss_0": "3.296", "loss_1": "0.138", "loss_2": "0.015", "accuracy": "0.48972", "wps": "6723.9", "ups": "1.08", "wpb": "6249.3", "bsz": "77.8", "num_updates": "22200", "lr": "3.46875e-05", "gnorm": "1.383", "train_wall": "183", "gb_free": "11.1", "wall": "14803"}
